dist/tools/buildsystem_sanity_check/check.sh

```diff
@@ -16,43 +16,29 @@
 : "${RIOTBASE:="$(cd "$(dirname "$0")/../../../" || exit; pwd)"}"
 
 : "${RIOTTOOLS:=${RIOTBASE}/dist/tools}"
+# shellcheck disable=SC1091
```

Comment:
Would be nice to add a commend here, what actually is disabled by this.

Reply:
Added and squashed.